corehq/toggles/__init__.py

```diff
@@ -944,6 +944,13 @@ def _ensure_valid_randomness(randomness):
     """
 )
 
+WEB_APPS_PERMISSIONS_VIA_GROUPS = StaticToggle(
```

Comment:
Might throw in a description pointing to the alternative, unless you did that on the page itself